x-pack/plugins/maps/public/classes/layers/vector_layer/vector_layer.tsx

```diff
@@ -635,14 +627,15 @@ export class VectorLayer extends AbstractLayer implements IVectorLayer {
       return;
     }
 
+    const meta = await this._getVectorSourceRequestMeta(syncContext.dataFilters, source, style);
     try {
       await this._syncSourceStyleMeta(syncContext, source, style);
       await this._syncSourceFormatters(syncContext, source, style);
       const sourceResult = await syncVectorSource({
         layerId: this.getId(),
         layerName: await this.getDisplayName(source),
         prevDataRequest: this.getSourceDataRequest(),
-        requestMeta: await this._getSearchFilters(syncContext.dataFilters, source, style),
+        requestMeta: meta,
```

Comment:
Can you still inline _getVectorSourceRequestMeta function call to avoid naming variable that is only passed into method call.